src/renderer/hub.html

```
@@ -29,7 +29,42 @@ <h1 class="title">BEACON</h1>
+
+        <div class="settings-inner">
+          <div class="settings-header">
+            <span class="settings-title">AI Coach</span>
+            <span id="key-badge" class="key-badge inactive">inactive</span>
+          </div>
+
+          <p class="settings-hint">Paste a free API key from <span class="hint-link">console.groq.com</span></p>
+
+          <div class="key-input-row">
+            <input
+              id="api-key-input"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)